src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -7373,28 +7373,7 @@ static void Main(string[] args)
         }
     }
 }";
-            var expected = @"class Test
-{
-    void method()
-    {
-        static void Main(string[] args)
-        {
-            int v = 0;
-            for(int i=0 ; i<5; i++)
-            {
-                v = NewMethod(v, i);
```

Comment:
so are you saying it will work if one of "}" not there? and whether it is nested or not seems not that relevant since the root issue we are trying to say is that extract method should work even in a broken code as long as it is not completely unreasonable. or let users know why it can't do it.